sdk/identity/azure-identity/src/main/java/com/azure/identity/AadCredentialBuilderBase.java

```diff
@@ -70,4 +70,31 @@ public T executorService(ExecutorService executorService) {
         this.identityClientOptions.setExecutorService(executorService);
         return (T) this;
     }
+
+    /**
+     * Sets whether to use an unprotected file specified by <code>cacheFileLocation()</code> instead of
+     * Gnome keyring on Linux. This is false by default.
+     *
+     * @param useUnprotectedFileOnLinux whether to use an unprotected file for cache storage.
+     *
+     * @return The updated T object.
```

Comment:
nit: to be consistent with the rest of the javadoc change this to 
`@return An updated instance of this builder with useUnprotectedFileOnLinux set as specified.`

Reply:
Updated